### DISCUSSION TITLE
닉네임 크기 변경

### DISCUSSION THREAD

**Discussion by strangestraw** [naverz/zepeto-studio-kor#329] [5. Q&A]
안녕하세요.
빌드잇으로 생성한 월드와 유니티에서 생성한 월드에서 카메라 위치에 따른 닉네임 크기 처리가 다른 것으로 보입니다.
빌드잇 월드에서는 카메라 거리에 따라 닉네임 사이즈가 변하는데,
유니티 월드에서는 카메라 거리에 무관하게 닉네임 사이즈가 유지됩니다.
유니티로 생성한 월드에서도 빌드잇의 닉네임과 동일하게 처리할 수 있는 방법이 있나요?
확인 부탁드립니다.
감사합니다.

**Comment by moondory77-naverz** [comment]
안녕하세요. 
현재 유니티 환경에서 개발한 제페토 월드의 경우, 
(카메라-캐릭터) 간의 거리조정에 대해, 닉네임 UI의 크기를 일정하게 유지하기 위한 자동 보정 Scaling 처리가 되어 있는데요.
그리고, 이는 기존 빌드잇을 통해 생성한 월드(Scaling 조정 없음)과 비교해 상이합니다.

다음 업데이트를 통해, Scaling 방식을 선택할 수 있는 옵션을 추가 제공드리는 방안을 검토해보겠습니다.
감사합니다.
